src/github_runner_image_builder/cli.py

```diff
@@ -140,6 +133,13 @@ def get_latest_build_id(cloud_name: str, image_name: str) -> None:
     help="EXPERIMENTAL: Proxy to use for external build VMs in host:port format (without scheme). "
     "Ignored if --experimental-external is not enabled",
 )
+@click.option(
```

Comment:
Added option --upload-cloud where the upload destination cloud is different from build cloud.